actions.yaml

```diff
@@ -6,3 +6,15 @@ rotate-session-key:
     Rotates the session key. 
     This will cause all users to be logged out of their current sessions (browser and CLI) and additionally, restart JIMM.
   additionalProperties: false
+
+set-host-key:
+  description: |
+    Sets the host key for JIMM and store it in a secret. The host key format is base64 encoded.
```

Comment:
The underlying formats accepted are OpenSSH format or PEM, let's add that info here.